var/spack/repos/builtin/packages/openfast/package.py

```diff
@@ -79,6 +79,15 @@ def cmake_args(self):
                 'ON' if '+cxx' in spec else 'OFF'),
         ])
 
+        # Make sure we use Spack's blas/lapack:
+        lapack_libs = spec['lapack'].libs.joined(';')
```

Comment:
Since you adding them later, It’s usually better to add ‘libs’ first and then do ‘joined()’ as addition also takes care of duplicates. 

Alternatively, judging from the input, you might as well provide just lapack and just blas.